### ISSUE TITLE
[Bug]: Unable to Install latest Version of Librewolf

### ISSUE THREAD

**Issue by sheepnaniganz** [ScoopInstaller/Extras#11013]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

librewolf

### Expected/Current Behaviour

Was unable to update due to errors so uninstalled and trying to install fresh and am getting an error. I am able to go to the url in the error and it does download so the url is valid and working.

### Steps to Reproduce

```console
ps> scoop install librewolf
WARN  Purging previous failed installation of librewolf.
ERROR 'librewolf' isn't installed correctly.
Removing older version (111.0.1-1).
'librewolf' was uninstalled.
Installing 'librewolf' (111.0.1-1) [64bit] from extras bucket
An error occurred while sending the request.
URL https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/111.0.1-1/./librewolf-111.0.1-1.en-US.win64-portable.zip is not valid
```

### Possible Solution

No idea

### Scoop and Buckets Version

```console
scoop --version
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
e144fba36 (HEAD -> master, origin/master, origin/HEAD) skaffold: Update to version 2.3.1

'java' bucket:
aed7f08f (HEAD -> master, origin/master, origin/HEAD) temurin8-nightly-jre: Update to version 8.0.372-6.0.202304101933

'main' bucket:
f062a2aed (HEAD -> master, origin/master, origin/HEAD) xxcopy: Use ScoopInstaller/Binary (#4657)

'nerd-fonts' bucket:
d7b401e (HEAD -> master, origin/master, origin/HEAD) SarasaGothic: Update to version 0.40.5

'nonportable' bucket:
8cda6d6 (HEAD -> master, origin/master, origin/HEAD) 10x-np: Update to version 1.0.9
'versions' bucket:
2eaa7e607 (HEAD -> master, origin/master, origin/HEAD) zig-dev: Update to version 0.11.0-dev.2546
```

### Scoop Config

```console
last_update           : 4/11/2023 9:13:10 PM
aria2-warning-enabled : False
aria2-enabled         : False
scoop_repo            : https://github.com/ScoopInstaller/Scoop
scoop_branch          : master
```

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.3.3
PSEdition                      Core
GitCommitId                    7.3.3
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by StarsbySea** [comment]
Can't reproduce. Change Your network and try again?
```
PS C:\Users\Kevin\scoop\buckets\extras\bin> scoop install librewolf
Installing 'librewolf' (111.0.1-1) [64bit] from extras bucket
librewolf-111.0.1-1.en-US.win64-portable.zip (102.3 MB) [=====================================================] 100%
Checking hash of librewolf-111.0.1-1.en-US.win64-portable.zip ... ok.
Extracting librewolf-111.0.1-1.en-US.win64-portable.zip ... done.
Running pre_install script...
Linking ~\scoop\apps\librewolf\current => ~\scoop\apps\librewolf\111.0.1-1
Creating shim for 'librewolf'.
Creating shortcut for LibreWolf (librewolf-portable.exe)
Persisting Profiles
Persisting LibreWolf\defaults\prefs
'librewolf' (111.0.1-1) was installed successfully!
```

I'm guessing it's an IPv6 problem or it's being blocked by Cloudflare's WAF, if that's possible run this command 
```
curl -v --Head https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/111.0.1-1/. /librewolf-111.0.1-1.en-US.win64-portable.zip
```
Then post the return results and see?

**Comment by HUMORCE** [comment]
Cannot reproduce.

```
> scoop install librewolf -u
WARN  Scoop is out of date.
Installing 'librewolf' (111.0.1-1) [64bit] from extras bucket
librewolf-111.0.1-1.en-US.win64-portable.zip (102.3 MB) [============================================================================] 100%
Checking hash of librewolf-111.0.1-1.en-US.win64-portable.zip ... ok.
Extracting librewolf-111.0.1-1.en-US.win64-portable.zip ... done.
Running pre_install script...
Linking ~\scoop\apps\librewolf\current => ~\scoop\apps\librewolf\111.0.1-1
Creating shim for 'librewolf'.
Creating shortcut for LibreWolf (librewolf-portable.exe)
Persisting Profiles
Persisting LibreWolf\defaults\prefs
'librewolf' (111.0.1-1) was installed successfully!
```

**Comment by HUMORCE** [comment]
So your firewall is configured incorrectly? try restore default settings for Windows defender firewall.

**Comment by StarsbySea** [comment]
https://support.microsoft.com/en-us/windows/firewall-network-protection-in-windows-security-aef9838b-d081-fd75-3b1b-e5fa794c003b
Try the link above to restore firewalls settings to default.

**Comment by Pkcarreno** [comment]
the same thing happens to me too, since version 111

```
❯ scoop install librewolf
Installing 'librewolf' (111.0.1-1) [64bit] from extras bucket
Starting download with aria2 ...
Download: 04/13 10:04:23 [ERROR] CUID#8 - Download aborted. URI=https://gitlab.com/api/v4/projects/44042130/packages/genDownload: Exception: [AbstractCommand.cc:351] errorCode=22 URI=https://gitlab.com/api/v4/projects/44042130/packages/geneDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: d2e993|ERR |       0B/s|C:/Users/pedro/scoop/cache/librewolf#111.0.1-1#https_gitlab.com_api_v4_projects_44042130_packages_generic_librewolf_111.0.1-1_._librewolf-111.0.1-1.en-US.win64-portable.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 22) HTTP response header was bad or unexpected
ERROR https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/111.0.1-1/./librewolf-111.0.1-1.en-US.win64-portable.zip
    referer=https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/111.0.1-1/./
    dir=C:\Users\pedro\scoop\cache
    out=librewolf#111.0.1-1#https_gitlab.com_api_v4_projects_44042130_packages_generic_librewolf_111.0.1-1_._librewolf-111.0.1-1.en-US.win64-portable.zip

ERROR & 'C:\Users\pedro\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\pedro\scoop\cache\librewolf.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.3 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=23480 --continue --summary-interval=0 --auto-save-interval=1

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=librewolf%40111.0.1-1%3a+download+via+aria2+failed
```

**Comment by sheepnaniganz** [comment]
Sorry I spoke too soon.

So for some reason it was just a coincidence that turning off the firewall worked for that install. Tried again later to install again and it went back to not working with the firewall on or off. Not sure why it worked and then stopped. 

I have checked all logging for my network and windows and everything is leaving the network fine and nothing is being blocked. IPv6 was turned off to make sure there was no issues there. 

Also it seems for some reason it mostly is happening with anything that uses gitlab.com. I tried to install `glab` and received the same error. I installed other packages that did not use gitlab and they worked fine, but those used only github. 

I also manually checked the download url from the bucket and am able to manually download through a browser and also through curl on powershell.

### Worked using Scoop

- Logseq
- Bind
- ungoogled-chromium
- speedtest
- conftest
- testdisk
- testcentric
- testmace

### Failed using scoop aria2 disabled

- Glab
- Firefox Nightly
- Firefox

```
>scoop install glab
WARN  Purging previous failed installation of glab.
ERROR 'glab' isn't installed correctly.
Removing older version (1.27.1).
'glab' was uninstalled.
Installing 'glab' (1.27.1) [64bit] from main bucket
An error occurred while sending the request.
URL https://gitlab.com/gitlab-org/cli/-/releases/v1.27.1/downloads/glab_1.27.1_Windows_x86_64.zip is not valid
```

### Worked using aria2 enabled
- Glab
- Firefox Nightly
- Firefox

```
scoop install glab
WARN  Purging previous failed installation of glab.
ERROR 'glab' isn't installed correctly.
Removing older version (1.27.1).
'glab' was uninstalled.
Installing 'glab' (1.27.1) [64bit] from main bucket
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 340fde|OK  |    24MiB/s|C:/Users/Sheepnaniganz/scoop/cache/glab#1.27.1#https_gitlab.com_gitlab-org_cli_-_releases_v1.27.1_downloads_glab_1.27.1_Windows_x86_64.zip
Download: Status Legend:
Download: (OK):download completed.
Checking hash of glab_1.27.1_Windows_x86_64.zip ... ok.
Extracting glab_1.27.1_Windows_x86_64.zip ... done.
Linking ~\scoop\apps\glab\current => ~\scoop\apps\glab\1.27.1
Creating shim for 'glab'.
'glab' (1.27.1) was installed successfully!
```

### Librewolf does not work using either method, but does work using winget and chocolatey and manually.

```
scoop install librewolf
Installing 'librewolf' (111.0.1-1) [64bit] from extras bucket
Starting download with aria2 ...
Download: 04/13 10:32:51 [ERROR] CUID#8 - Download aborted. URI=https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/111.0.1-1/./librewolf-111.0.1-1.eDownload: Exception: [AbstractCommand.cc:351] errorCode=22 URI=https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/111.0.1-1/./librewolf-111.0.1-1.enDownload:   -> [HttpSkipResponseCommand.cc:239] errorCode=22 The response status is Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 4cbc28|ERR |       0B/s|C:/Users/Sheepnaniganz/scoop/cache/librewolf#111.0.1-1#https_gitlab.com_api_v4_projects_44042130_packages_generic_librewolf_111.0.1-1_._librewolf-111.0.1-1.en-US.win64-portable.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 22) HTTP response header was bad or unexpected
ERROR https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/111.0.1-1/./librewolf-111.0.1-1.en-US.win64-portable.zip
    referer=https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/111.0.1-1/./
    dir=C:\Users\Sheepnaniganz\scoop\cache
    out=librewolf#111.0.1-1#https_gitlab.com_api_v4_projects_44042130_packages_generic_librewolf_111.0.1-1_._librewolf-111.0.1-1.en-US.win64-portable.zip

ERROR & 'C:\Users\Sheepnaniganz\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\Sheepnaniganz\scoop\cache\librewolf.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.3 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=5988 --continue --summary-interval=0 --auto-save-interval=1

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=librewolf%40111.0.1-1%3a+download+via+aria2+failed

>scoop config aria2-enabled false
'aria2-enabled' has been set to 'false'

>scoop install librewolf
WARN  Purging previous failed installation of librewolf.
ERROR 'librewolf' isn't installed correctly.
Removing older version (111.0.1-1).
'librewolf' was uninstalled.
Installing 'librewolf' (111.0.1-1) [64bit] from extras bucket
An error occurred while sending the request.
URL https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/111.0.1-1/./librewolf-111.0.1-1.en-US.win64-portable.zip is not valid
```

If there is any other testing I can do let me know.

**Comment by HUMORCE** [comment]
Provide your Scoop configuration here, `scoop config` or `cat ~\.config\scoop\config.json`

**Comment by HUMORCE** [comment]
Temporarily closed, feel free to reopen this issue.

**Reasons:**
- Cannot reproduce.
- No response and additional information
